moveit_ros/planning_interface/test/dual_arm_robot_state_update.py

```diff
@@ -32,11 +33,13 @@ def test(self):
         rocket_pose.pose.position.x += 0.05
 
         groot_pose = self.groot_group.get_current_pose("groot_tool0")
-        rocket_pose.pose.position.z -= 0.05
-        rocket_pose.pose.position.x -= 0.05
+        groot_pose.pose.position.z -= 0.05
```

Comment:
it's slightly misleading to "hide" non formatting related changes like this in a commit titled "Fix format"

also if you install and enable pre-commit, this should be fixed automatically

Reply:
Yeah, sorry, I re-organized the commits, just two the one with the test and the one reverting the issue being addressed here.